src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/config/InterceptorConfig.java

```diff
@@ -30,10 +34,17 @@ public InterceptorConfig(LoggingInterceptor loggingInterceptor, CustomUserAuthen
      */
     @Override
     public void addInterceptors(@NonNull InterceptorRegistry registry) {
+        registry.addInterceptor(transactionInterceptor())
```

Comment:
logging interceptor should always come first so it will log everything after

Reply:
Reordered the config.